sampleproject/components/fragment.py

```diff
@@ -0,0 +1,156 @@
+from django_components import Component, types
+
+
+# Common logic for all FragmentBase components
+class _FragmentBase(Component):
+    def get(self, request):
+        return self.render_to_response()
```

Comment:
I think it's probably easier to follow is this is copied across all FragmentBaseJS classes.